app/models/manageiq/providers/redfish/inventory/parser/physical_infra_manager.rb

```diff
@@ -28,5 +29,36 @@ def physical_servers
         )
       end
     end
+
+    def physical_server_details
+      # TODO(tadeboro): There is no similar data in Redfish service, so
+      # mapping will need to be quite sophisticated if we would like to get
+      # more info into database.
+      collector.physical_server_details.each do |d|
+        server = persister.physical_servers.lazy_find(d[:server_id])
+        details = persister.physical_server_details.build(server)
+        details.assign_attributes(
```

Comment:
Actually you don't need to assign_attribtues when using build, just pass this hash into build and pass in the manager ref.
So I think for this it'd be:
```
persister.physical_server_details.build(
  :resource    => server,
  :contact     => "",
  :description => "",
  ...
)
```

Reply:
Sorry for the noise, I pushed the partial edit. Things should look better now.